gaeul/source/source-application.c

```diff
@@ -399,9 +404,10 @@ gaeul_source_application_command_line (GApplication * app,
 
     passphrase = g_settings_get_string (ssettings, "passphrase");
     pbkeylen = g_settings_get_enum (ssettings, "pbkeylen");
+    use_hw_vaapi_decode = g_settings_get_boolean (ssettings, "use-hw-vaapi-decode");
 
     if (!gaeguli_nest_start (nest, stream_id, target_uri, video_codec, bitrate,
-            passphrase, pbkeylen)) {
+            passphrase, pbkeylen, use_hw_vaapi_decode)) {
```

Comment:
The pipeline is already instantiated so we can set `"use-hw-vaapi-decode"` right here instead of adding another parameter to `gaeguli_nest_start()`. Makes things simpler.